package.json

```diff
@@ -26,10 +26,10 @@
   },
   "dependencies": {
     "bluebird": "^3.0.0",
+    "buffer-equal-constant-time": "^1.0.1",
```

Comment:
Buffer has a built in `equals` method, no need for this library: https://nodejs.org/api/buffer.html#buffer_buf_equals_otherbuffer